__tests__/app.test.js

```diff
@@ -284,4 +284,48 @@ describe('POST/api/articles/:article_id/comments', () => {
         expect(response.body.msg).toBe("bad request")
       })
     });
+    test('STATUS - 400: should return appropriate error when passed invalid key in body', () => {
+      return request(app)
+      .post("/api/articles/2/comments")
+      .send({username: "rogersop", comment: "Very interesting"})
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("bad request")
+      })
+    });
+    test('STATUS - 400: should return appropriate error when passed a body with too many keys', () => {
+      return request(app)
+      .post("/api/articles/2/comments")
+      .send({username: "rogersop", comment: "Very interesting", age: 40})
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("bad request")
+      })
+    });
+    test('STATUS - 400: should return appropriate error when not passed a body', () => {
+      return request(app)
+      .post("/api/articles/2/comments")
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("bad request")
+      })
+    });
+    test('STATUS - 404: should return appropriate error when passed an invalid article ID ', () => {
```

Comment:
_Really_ picky here, it is a 404, but its not an invalid id, its valid but not there.